server/lib/tuist/environment.ex

```diff
@@ -215,7 +225,7 @@ defmodule Tuist.Environment do
     if dev_use_remote_storage?() do
       get([:aws, :endpoint], secrets) || get([:s3, :endpoint], secrets)
     else
-      "http://localhost:9095"
+      get([:local_s3_endpoint], secrets) || "http://localhost:9095"
```

Comment:
we need to be able to run the server and a `mix` instance side-by-side as the agent is calling the localhost server and that can't be done from a single elixir process. The minio port was conflicting, so I added a way to configure it, so we can run `TUIST_WEB=0 TUIST_LOCAL_S3_ENDPOINT="http://localhost:9096" iex -S mix` while `mise run server:dev` is already running